lib/matplotlib/lines.py

```diff
@@ -1329,6 +1329,7 @@ def get_solid_joinstyle(self):
     def set_dash_capstyle(self, s):
         """
         How to draw the end caps if the line is `~Line2D.is_dashed`.
+        The default capstyle is set by ``rcParams['lines.dash_capstyle']``.
```

Comment:
```suggestion

        The default capstyle is :rc:`lines.dash_capstyle`.
```
We have a special directive for that. And there has to be a blank line after the first summary sentence in docstrings.

Reply:
fixed.